shell/platform/embedder/embedder.h

```diff
@@ -2510,6 +2510,25 @@ FLUTTER_EXPORT
 FlutterEngineResult FlutterEngineScheduleFrame(FLUTTER_API_SYMBOL(FlutterEngine)
                                                    engine);
 
+//------------------------------------------------------------------------------
+/// @brief      Schedule a callback to be called after the next frame is drawn.
+///             This callback is made on an internal engine managed thread and
```

Comment:
(or an embedder managed thread if custom task runners are specified)

Reply:
Ah, perhaps my original comment is incorrect. This runs on the raster thread, which is created by the embedder (and thus also managed by the embedder?). Should I just say it runs on the raster thread?

Reply:
I switched this to say raster thread. I'll leave this conversation open for now, let me know if you'd prefer I go with your original message!